RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# Whats New in 16.5.71
+
+* Add missing requires to chef/policy_builder/dynamic.
+* Check for full name in addition to short name for `homebrew_package` resource. `homebrew_package 'homebrew/core/vim'` and `homebrew_package 'vim'` styles should now both work correctly.
+
```

Comment:
You might want to mention we improved the license acceptance flow to give you helpful information if you entered an incorrect string